### PR TITLE
Capture command line args (used by Equalizer during launch)

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by hernando** [comment]
If your idea is to test it with RTNeuron without messing with EQ_NODE_SATTR_LAUNCH_COMMAND it won't be so straight forward. The reasons are:

* I also add some environmental variables to the command using env.
* I quote one argument that accepts regular expressions because some characters confuse bash (and the quoting that was originally used is lost). As a matter of fact, maybe all arguments should be quoted just in case.
* I convert arguments starting with ./ and ../ to absolute paths. That could be avoided is Equalizer changes the remote working directory to the same location from where you launch the application.

**Comment by eile** [comment]
* env variables: Which ones? Can we generalize this feature?
* quote: The Eq code quotes each arg individually: ```...eqPly' '--eq-config' 'examples/configs/2-node.eqc' '--deine-mutter' -- --eq-client  --co-launch '1#4...```
* workdir: It does, see ConfigParams::getWorkDir.

**Comment by hernando** [comment]
I noticed 2 after I got the notification of eyescale/Equalizer/#578. For 3, is the working directory setup automatically? As for 1, setting up a list of prefixes of vars to be transmitted can be a good idea.

**Comment by hernando** [comment]
I've checked the working directory is passed on to the clients, but I don't find any call to setcwd. Is this something the application has to do manually?

**Comment by eile** [comment]
> is the working directory setup automatically?
Yes: https://github.com/Eyescale/Equalizer/blob/master/eq/init.cpp#L132

**Comment by eile** [comment]
> Is this something the application has to do by itself?
No, done here: https://github.com/Eyescale/Equalizer/blob/master/eq/client.cpp#L254

**Comment by hernando** [comment]
OK, but https://github.com/Eyescale/Equalizer/blob/master/eq/global.cpp#L56 is not really changing it.

**Comment by hernando** [comment]
Forget my comment, I just saw chdir is in client.cpp:255. However that doesn't quite work for me because I have to change it before initLocal is called.

File: co/localNode.h [hunk 1]
```diff
@@ -453,6 +453,9 @@ class LocalNode : public lunchbox::RequestHandler, public Node,
      */
     CO_API bool inCommandThread() const;
 
+    /** @return the non-Collage command line options passed to initLocal(). */
```
Comment (hernando) [inline co/localNode.h]: missing version

**Comment by hernando** [comment]
You haven't updated the changelog and the minor version number, other than that it's OK.

File: co/localNode.h (hunk 1 above)
Comment (eile) [inline co/localNode.h]: I|we stopped doing it since we don't do it in the other projects either.

**Comment by eile** [comment]
Do you think changelog is needed? I figured this is not worth documenting.

File: co/localNode.h (hunk 1 above)
Comment (hernando) [inline co/localNode.h]: I think it was a good idea, what you have stopped in other projects?

**Comment by hernando** [comment]
You can judge better than me, and I don't know which level of detail is consistent with the log history.
But the version should be bumped, shouldn't it?
+1 otherwise

**Comment by eile** [comment]
Yes, version bumped now.

**Pull request merged**